packages/astro/src/vite-plugin-utils/index.ts

```diff
@@ -0,0 +1,20 @@
+import type { AstroConfig } from '../@types/astro';
+import { appendForwardSlash } from '../core/path.js';
+
+interface AstroPluginOptions {
+	config: AstroConfig;
+}
+export function getFileInfo(id: string, config: AstroConfig) {
+	const sitePathname = appendForwardSlash(
+		config.site ? new URL(config.base, config.site).pathname : config.base
+	);
+
+	const fileId = id;
```

Comment:
nit: any reason to return `fileId` if it's just the `id` passed in?

Reply:
Nope, was just copy + pasting. Can rename some of these vars.

Reply:
Ah, this was supposed to strip query params. Fixed it!